examples/inverse_folding/README.md

```diff
@@ -0,0 +1,170 @@
+# Inverse folding with ESM-IF1
+
+The ESM-IF1 inverse folding model is built for predicting protein sequences 
+from their backbone atom coordinates. We provide scripts here 1) to sample sequence 
+designs for a given structure and 2) to score sequences for a given structure. 
+
+Trained with 12M protein structures predicted by AlphaFold2, the ESM-IF1
+model consists of invariant geometric input processing layers followed by a
+sequence-to-sequence transformer, and achieves 51% native sequence recovery on
+structurally held-out backbones with 72% recovery for buried residues.
+The model is also trained with span masking to tolerate missing backbone
+coordinates and therefore can predict sequences for partially masked structures.
+
```

Comment:
Already write out a paragraph refering readers to the paper for more details - just a placeholder link!